app/pipeline/transcription_ingestion_pipeline.py

```
@@ -0,0 +1,250 @@
+import threading
```

Ruff (0.8.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)